README.md

```diff
@@ -159,6 +159,16 @@ not present in the project's `gradle.properties`:
 
 -------
 
+#### Applying to an application module
+
+`Paparazzi.inflate` would return `null` because the application module merges resources into a binary format which is not supported yet.
```

Comment:
Users don't need to know this because your change means they'll never see it

Reply:
Removed the technical detail in https://github.com/cashapp/paparazzi/pull/301/commits/7cf0f8032f84c05ea421380b96d23514b44a7bfe, but the rest is useful because it is a known limitation that it doesn't work on apps, right?

Reply:
Yep